src/message/MessageListContainer.js

```diff
@@ -105,6 +109,16 @@ class MessageListContainer extends PureComponent<Props> {
     );
   };
 
+  handleUrlLongPress = (src: string) => {
+    if (isUrlAnImage(src)) {
+      const { showActionSheetWithOptions, auth } = this.props;
+      openLightboxActionSheet({ showActionSheetWithOptions, src, auth });
+      return;
+    }
+    Clipboard.setString(src);
```

Comment:
Also export `Clipboard` related work via a util method (like `showToast`).
Advantages: 
Let's say in future RN changes API then we can easily upgrade by updating in single method. (Component will be untouched)

Reply:
That can be done. In a separate PR or here?

Reply:
Create a method for it in a separate commit (can be done in this PR too) and use it in another commit (along with the changes in this PR)